.bithoundrc

```diff
@@ -0,0 +1,13 @@
+{
+  "critics": {
+    "lint": {
+      "engine": "eslint"
+    },
+    "wc": {
+      "limit": 30000
```

Comment:
I think 30000 for this value is too big, 5000 will be more than enough in our case.

Reply:
5000 isn't 5000 lines, it's 5000 bytes

Reply:
I know that, and at the moment 5000 is more than enough:
```bash
$ wc -c sandboxed-fs.js 
3834 sandboxed-fs.js
```